packages/nx/src/plugins/js/index.ts

```diff
@@ -91,7 +95,10 @@ export const createDependencies: CreateDependencies = (
     parsedLockFile.externalNodes
   ) {
     const lockFilePath = join(workspaceRoot, getLockFileName(packageManager));
-    const lockFileContents = readFileSync(lockFilePath).toString();
+    const lockFileContents =
+      packageManager !== 'bun'
+        ? readFileSync(lockFilePath).toString()
+        : execSync(lockFilePath).toString();
```

Comment:
Shouldn't it be `bun bun.lockb` as the command?

Reply:
It's checking on the actual package manager and not the lock file.  I thought it would be more readable to check the package manager saying we already have the reference. Otherwise its checking the path

Reply:
Running just `bun.lockb` throws an error. I'm not sure I understand the comment above, but I believe we still need to parse it with `bun bun.lockb` as @JamesHenry suggested to be able to extract the hash (or hash it altogether) unless there is another way to get that hash out of the lockfile.

Reply:
bun.lockb is self executing (at least on linux) Happy to change this to bun bun.lockb as it makes more sense

Reply:
It doesn't seem self-executing on mac OS, so please use the `bun bun.lockb` as that should work across the OSs.